src/lib.rs

```diff
@@ -43,7 +43,7 @@ lazy_static::lazy_static! {
 #[cfg(feature = "render_element")]
 fn next_id() -> usize {
     let mut ids = EGUI_IDS.lock().unwrap();
-    debug_assert!(!ids.len() == usize::MAX);
+    debug_assert!(!(ids.len() == usize::MAX));
```

Comment:
Why was this no `!=` in the first place :sweat_smile: 

```suggestion
    debug_assert!(ids.len() != usize::MAX);
```

Reply:
yeah that is even better :)